package.json

```diff
@@ -48,6 +48,7 @@
     "json-loader": "^0.5.4",
     "less": "^2.7.1",
     "less-loader": "^4.0.5",
+    "npm": "^5.8.0",
```

Comment:
Why is this in package.json if we're going to install it via `npm install` in Travis anyway?

The proper way to specify version is still in `engines` key, whether that works in Travis or not. https://docs.npmjs.com/files/package.json#engines

Reply:
... if Travis is using this, then why is `npm install npm` necessary?

Reply:
It is in `devDependencies`, because developers need to use it when generating their `package-lock.json` , or they will generate a junky one which includes optional dependencies, even if `--no-optional` is used.

After the first uncached build, npm 5.8 will be cached and used at the beginning of the next build.

Travis' issue says there is a difference of meaning with `engines` between "install" and "build".  Not sure how relevant that is in this type of repo.  The latest npm is usable to do an install -- it is only additions of new dependencies where the older version is needed.

Reply:
Added to engines.